File: VSExplorer/bin/Debug/Templates-Data/ItemTemplates/CSharp/Web/General/1033/Handler/Handler.ashx.cs
using System;
using System.Collections.Generic;
$if$ ($targetframeworkversion$ >= 3.5)using System.Linq;
$endif$using System.Web;

namespace $rootnamespace$
{
    /// <summary>
    /// Summary description for $classname$
    /// </summary>
    public class $classname$ : IHttpHandler {

        public void ProcessRequest (HttpContext context) {
            context.Response.ContentType = "text/plain";
            context.Response.Write("Hello World");
        }
     
        public bool IsReusable {
            get {
                return false;
            }
        }
    }
}

File: VSExplorer/bin/Debug/Templates-Data/ItemTemplates/VisualBasic/Web/General/1033/Handler/Handler.ashx.vb
Imports System.Web
Imports System.Web.Services

Public Class $classname$
    Implements System.Web.IHttpHandler

    Sub ProcessRequest(ByVal context As HttpContext) Implements IHttpHandler.ProcessRequest

        context.Response.ContentType = "text/plain"
        context.Response.Write("Hello World!")

    End Sub

    ReadOnly Property IsReusable() As Boolean Implements IHttpHandler.IsReusable
        Get
            Return False
        End Get
    End Property

End Class